stubs/paramiko/paramiko/transport.pyi

```diff
@@ -26,6 +26,9 @@ class _KexEngine(Protocol):
 class Transport(Thread, ClosingContextManager):
     active: bool
     hostname: str | None
+    server_extensions: dict[str, str]
```

Comment:
```suggestion
    server_extensions: dict[str, bytes]
```
I think this might be a slightly different type based on https://github.com/paramiko/paramiko/blob/main/paramiko/transport.py#L2818

Reply:
Thanks for the information 👍🏻 
I have pushed the new type.